kernel/content/restore.php

```diff
@@ -134,7 +134,6 @@
     $version->store();
 
     $object->restoreObjectAttributes();
-
```

Comment:
I guess we can skip this, and if we want to reduce the changes of the two PR's in combination there should afaik be a extra new line before `$db->commit();` :P